internal/integration/go.mod

```
@@ -5,8 +5,8 @@ go 1.17
 replace ariga.io/atlas => ../../
 
 require (
-	ariga.io/atlas v0.0.0
-	entgo.io/ent v0.9.2-0.20211116120827-6954eb3a182a
+	ariga.io/atlas v0.3.5-0.20220215131223-8043663b4223
```

```suggestion
	ariga.io/atlas v0.0.0
```
It's using `replace` above.